tests/Functional/SymfonyProxyClientTest.php

```diff
@@ -69,7 +69,11 @@ public function testRefresh()
         $this->getProxyClient()->refresh('/symfony.php/cache')->flush();
         usleep(100);
         $refreshed = $this->getResponse('/symfony.php/cache');
-        $this->assertGreaterThan((float) $response->getBody(true), (float) $refreshed->getBody(true));
+
+        $originalTimestamp = (float)(string) $response->getBody();
+        $refreshedTimestamp = (float)(string) $refreshed->getBody();
+        
+        $this->assertGreaterThan($originalTimestamp, $refreshedTimestamp);
```

Comment:
this is basically the same code as above for nginx, but it looks differently. could we even unify that? though not really part of this PR, but lets take note somewhere